pkg/skaffold/build/plugin/plugin.go

```diff
@@ -88,14 +92,23 @@ func NewPluginBuilder(cfg *latest.BuildConfig, opts *config.SkaffoldOptions) (sh
 		builders[p] = pluginBuilder
 	}
 
+	conn, err := grpc.Dial(opts.RPCPort, grpc.WithInsecure())
+	if err != nil {
+		logrus.Warnf("unable to open gRPC connection to skaffold [%s]: events will not be handled properly!", err.Error())
+		// os.Exit(1)
```

Comment:
Also, as discussed, let's rework this through the Init() method of the plugins and a transparent way of branching between plugin and non-plugin mode in `event.Handle`